source/common/router/tls_context_match_criteria_impl.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "envoy/api/v2/route/route.pb.h"
+#include "envoy/router/router.h"
+
+namespace Envoy {
+namespace Router {
+
+class TlsContextMatchCriteriaImpl;
+using TlsContextMatchCriteriaImplConstPtr = std::unique_ptr<const TlsContextMatchCriteriaImpl>;
```

Comment:
Only include these if you need this type, no need to forward declare.

Reply:
removed